backend/scripts/build_db_script.py

```diff
@@ -0,0 +1,104 @@
+import json
+import datetime
+import pymongo
+import os
+from dotenv import load_dotenv
+
+# Loading env variables file
+load_dotenv('../.env')
+
+DIST = "D"  # Prefix in every node distance key [eg. D1, D2]
+REV_DIST = "RD"
+INPUT_FILENAME = "fdg_input_file.json"

```

Comment:
if this file name cannot change, add a comment indicating this

Reply:
No, the user can change the filename. Maybe I should rename it to `INPUT_FILE_PATH` and add a comment for better clarity.